src/eureka/S3_data_reduction/s3_reduce.py

```diff
@@ -340,6 +340,19 @@ def reduce(eventlabel, ecf_path=None, s2_meta=None, input_meta=None):
                 # Manually mask regions [colstart, colend, rowstart, rowend]
                 if hasattr(meta, 'manmask'):
                     data = util.manmask(data, meta, log)
+                
+                # Mask uncalibrated BG region for NIRSpec observations
+                # Code used for generating a calibrated stellar spectrum
+                if meta.convert_to_e is False and meta.inst == 'nirspec':
+                    cutoff = 1e-4
+                    log.writelog("  Masking manually identified bad pixels...",
+                                 mute=(not meta.verbose))
+                    boolmask = np.abs(data.flux.data) > cutoff
+                    log.writelog(f"    Masking {np.sum(boolmask.data)} " + 
+                                 "pixels.", mute=(not meta.verbose))
+                    data['flux'].data = np.where(np.abs(data.flux.data) > 
+                                                 cutoff, 0,
+                                                 data.flux.data)
```

Comment:
Why set them to zero instead of updating the mask array?

Reply:
The uncalibrated BG region is everything >8 pixels from the trace.  I originaly tried masking those pixels, but Eureka! started throwing way too many errors for that to be a reliable solution.  I ended up picking this solution, which effectively converts the uncalibrated BG region to MJ/sr (by setting the values to 0 since GLBS already happened in Stage 1).

Reply:
But there is no check that GLBS has happened in Stage 1, and it would seem entirely possible to me that someone would not want to run GLBS or want to investigate not doing it

Reply:
Going back to my initial PR...
``I added some code that masks the uncalibrated part of the NIRSpec 2d array. It's quick and dirty, but it works. Once I get more experience with other instruments, I'll certainly want to make changes/enhancements. It's just not worth devoting more time until I know how the other instrument will work.``

If someone decides not to use GLBS in Stage 1 and wants calibrated NIRSpec pectra, then their results will not be accurate.  For NIRSpec, there's just no way of accurately subtracting the BG after performing the Photom step in Stage 2.